### PR TITLE
LPS-197716 | Test Fix

### PR DESCRIPTION
**Pull request by victorhcunha**
[LPS-197716](https://liferay.atlassian.net/browse/LPS-197716)

### CODE REVIEW

**Comment by victorhcunha** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectPortlet.testcase [hunk 1]
```diff
@@ -799,7 +799,21 @@ definition {
...
+
+		ClickNoError(locator1 = "ObjectPortlet#DELETE_ENTRY_BUTTON");
+
+		AssertTextEquals(
+			locator1 = "WorkflowAllItems#DETAIL_PAGE_TITLE",
+			value1 = "Delete Entry");
+
+		AssertTextEquals(
+			locator1 = "WorkflowDefinition#DIALOG_BOX_BODY",
+			value1 = "It may affect many records. Are you sure you want to delete this entry?");
```
Comment (victorhcunha) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectPortlet.testcase:814]: None of the other tests cover the new modal, so as not to be onerous to cover it by the macro (impacting everyone), this specific test must cover it. This test has the property `testray.component.names = "Frontend Data Set"`

**Comment by victorhcunha** [comment]
ci:test:sf

**Review by andreefarias — APPROVED** [review]
LGTM! 🎉

**Comment by andreefarias** [comment]
Hey @timpak,

I reviewed this PR and it's good to me. I'll take the freedom to forward this PR, because this fix is cause many failures on routine.

If you had some suggestion, feel free to request changes. Victor can check if necessary.

cc: @victorhcunha

**Comment by andreefarias** [comment]
ci:forward

**Pull request closed without merging**